util/design/lib/LcStEnc.py

```diff
@@ -292,7 +292,7 @@ def __init__(self, config):
             strong_random.load(config['entropy_buffer'])
         else:
             # Re-initialize with seed to make results reproducible.
-            strong_random.generate_from_seed(
+            strong_random.unsafe_generate_from_seed(
```

Comment:
Would `unsecure` be more appropriate here? It's safe to use (i.e. doesn't crash or anything) but not cryptographically secure ;)...

Reply:
sure I'm OK with `unsecure`, @moidx [suggested](https://github.com/lowRISC/opentitan/pull/18346#discussion_r1185631068) `unsafe`, I'm OK with either.

Reply:
I agree that `unsecure` is more precise

Reply:
`unsecure` it is! fixed.